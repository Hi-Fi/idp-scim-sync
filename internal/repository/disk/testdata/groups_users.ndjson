{"items":1,"hashCode":"123456789","group":{"id":"1","name":"group 1","email":"group.1@mail.com","hashCode":"123456789"},"resources":[{"id":"1","name":{"familyName":"1","givenName":"user"},"displayName":"user 1","active": true, "email":"user.1@mail.com","hashCode":"123456789"}]}
{"items":1,"hashCode":"987654321","group":{"id":"2","name":"group 2","email":"group.2@mail.com","hashCode":"987654321"},"resources":[{"id":"2","name":{"familyName":"2","givenName":"user"},"displayName":"user 2","active": true, "email":"user.2@mail.com","hashCode":"987654321"}]}