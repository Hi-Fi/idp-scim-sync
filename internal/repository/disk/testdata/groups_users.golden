{"items":2,"hashCode":"123456789","resources":[{"items":1,"groups":{"id":"1","name":"group 1","email":"g.1@mail.com","hashCode":"123456789"},"hashCode":"123456789","resources":[{"id":"1","name":{"familyName":"1","givenName":"user"},"displayName":"user 1","active": true, "email":"user.1@mail.com","hashCode":"123456789"}]},{"items":1,"groups":{"id":"2","name":"group 2","email":"g.2@mail.com","hashCode":"987654321"},"hashCode":"987654321","resources":[{"id":"2","name":{"familyName":"2","givenName":"user"},"displayName":"user 2","active": true, "email":"user.2@mail.com","hashCode":"987654321"}]}]}